src/Barometer.cpp

```diff
@@ -1,156 +1,55 @@
 #include <Barometer.h>
 #include <GOST4401_81.h>
 
-// Constructors //////////////////////////////////////////////////////
+LPS331::LPS331(uint8_t slaveAddress)
+    : BaseIMU(slaveAddress) { }
 
-LPS331::LPS331(uint8_t addr) { address = addr; }
-
-// Public Methods ////////////////////////////////////////////////////
-
-// sets or detects slave address; returns bool indicating success
 void LPS331::begin(TwoWire& wire) {
     _wire = &wire;
     _wire->begin();
-    writeReg(LPS331_CTRL_REG1, 0b11100000);
-    delay(100);
+    _ctrlReg1 |= (1 << 7);
+    _ctrlReg1 |= (1 << 6);
+    _ctrlReg1 |= (1 << 5);
```

Comment:
_ctrlReg1 |= 0b11100000 ?

Reply:
This is data from older commit.
See the latest commit:
https://github.com/amperka/Troyka-IMU/pull/19/commits/66c569ac397a777327247eff8744b723ae126527